src/react-apps/applications/ux-editor/src/types/global.ts

```diff
@@ -64,6 +64,7 @@ declare global {
     dataModelBinding?: string;
     textResourceId?: string;
     customType?: string;
+    simplified?: boolean;
```

Comment:
do you need this on both the ICreateFormComponent and IFormAdressComponent?

Reply:
Removed from this, and the boolean is only on IFormAdressComponent.
Also fixed my typo IFormAdressComponent -> IFormAddressComponent.